frontend/src/core/user/components/FileDownload.js

```diff
@@ -0,0 +1,56 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from 'fluent-react';
+
+import type { NavigationParams } from 'core/navigation';
+
+
+type Props = {|
+    parameters: NavigationParams,
+|};
+
+
+/*
+ * Render a Sign Out link.
+ */
+export default class SignOut extends React.Component<Props> {
+    downloadTranslations = () => {
+        document.getElementById('download-form').submit();
```

Comment:
I'm getting an interesting `flow` error here:
```
Cannot call document.getElementById(...).submit because:
 • property submit is missing in HTMLElement [1].
 • property submit is missing in null [2].
```

Reply:
Yeah, Flow isn't working very well with DOM elements. You can either work around the error but checking that what's returned by `getElementById` is of the correct type (but meeeh) or simply ignore this error by putting `// $FLOW_IGNORE` on the previous line, ideally with a short comment explaining why we ignore this (in this case, we are pretty sure the element will exist).

Reply:
Actually, you should use `refs` in React in order to reach a DOM element of your component. See https://reactjs.org/docs/refs-and-the-dom.html

Reply:
You could also put your `button` inside the `form` and avoid this function entirely.